### PR TITLE
Mono-Service ready for modules

### PR DESCRIPTION
**Pull request by hendrikebbers**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/config/FileNumbers.java [hunk 1]
```diff
@@ -13,13 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.hedera.services.config;
-
-import static com.hedera.services.config.EntityNumbers.UNKNOWN_NUMBER;
+package com.hedera.node.app.service.mono.config;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/config/FileNumbers.java:17]: Can we get the import back ?

**Review by Neeharika-Sompalli — COMMENTED** [review]
Something changed. Platform is not becoming `ACTIVE`. Need to look at that.

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/config/FileNumbers.java (hunk 1 above)
Comment (hendrikebbers) [reply hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/config/FileNumbers.java:17]: will do :)

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by tinker-michaelj — COMMENTED** [review]
LGTM!

**Review by tinker-michaelj — APPROVED** [review]
🥂

**Pull request merged**